scripts/vagrant/Vagrantfile

```diff
@@ -3,8 +3,10 @@
 
 Vagrant.configure("2") do |config|
 
-  config.vm.synced_folder "../..", "/go/src/github.com/ligato/networkservicemesh"
-  config.vm.synced_folder ".", "/vagrant"
+  config.vm.synced_folder "../..", "/go/src/github.com/ligato/networkservicemesh",
```

Comment:
Why we need this change here?
Can you put it in a separate PR and explain.

Reply:
Without it, one of variants is ignored, in my case it is /vagrant and images are not published, happens sometimes.